mcodingbot/plugins/donors.py

```diff
@@ -82,6 +88,15 @@ async def _is_donor(member: hikari.Member) -> bool:
 async def _update_donor_role(
     member: int | hikari.Member, is_donor: bool
 ) -> None:
+    if not CONFIG.mcoding_server:
+        LOGGER.warning(
+            "Could not update donor roles because mcoding server id was not"
+            " provieded. Is this intended?"
+        )
```

Comment:
Rather than warning here, maybe it should be done in the start function of the bot? Here it should just return. Thinking this because right now, this only warns if you use the set-donor-status command, but not for the loop (because the loop/task already returns)